include/boost/histogram/algorithm/empty.hpp

```diff
@@ -0,0 +1,38 @@
+// Copyright 2019 Hans Dembinski and Henry Schreiner
```

Comment:
Since you wrote the code, I think only you should be listed here. I will add my name if I make changes.

Reply:
Okay, I didn't know what the right thing to put here was.